tests/e2e/specs/out-of-game/health.spec.js

```diff
@@ -0,0 +1,38 @@
+const { version: pkgVersion } = require('../../../../package.json');
+const sinon = require("sinon")
+const Game = require('../../../../api/models/Game')
+
+let stub;
+describe('Health Page', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  afterEach(() => {
+    if (stub) {
+      stub.restore();
+    }
+  });
+
+  it('is available if db responds', () => {
+    cy.request('/health').then((res) => {
+      expect(res).property('status').to.equal(200);
+      expect(res.body).property('alive').to.eq(true);
+    });
+  });
+
+  it.skip('is not available when db throws any error', () => {
+    stub = sinon.stub(Game, "count").throws();
+    cy.request('/health').then((res) => {
+      expect(res).property('status').to.equal(200);
+      expect(res.body).property('alive').to.eq(false);
+    });
+  });
```

Comment:
This didn't work right? I'd expect an issue because Game won't be globally available outside of backend code. Let's remove it and Sinon altogether since we don't currently have a working use case
```suggestion
```

Reply:
Removed all, kept the try/catch update.